README.md

```diff
@@ -932,54 +933,47 @@ Google additional consent is supported in our mobile SDKs and is stored in the `
 
 ## Global Privacy Platform Multi-State Privacy (MSPS) Support for OTT
 
-Starting with version 7.3.0, if your configuration contains a ccpa campaign, it will automatically set GPP data. Unless configured otherwise, the following MSPA attributes will default to:
+Starting with version 7.6.0, if your configuration contains a ccpa campaign, it will automatically set GPP data. 
 
-- MspaCoveredTransaction: `NO`
-- MspaOptOutOptionMode: `NOT_APPLICABLE`
-- MspaServiceProviderMode: `NOT_APPLICABLE`
+## Migrating Existing CCPA Consent to Multi-State Privacy (MSPS) Consent
```

Comment:
I reorganized the section below. Please let me know if this makes sense:

## Transfer opt-in/opt out preferences from U.S. Privacy (Legacy) to U.S. Multi-State Privacy

When migrating a property from the U.S. Privacy (Legacy) campaign to U.S. Multi-State Privacy campaign, the SDK will automatically detect previously set end-user opt-in/opt-out preferences for U.S. Privacy (Legacy) and have that transferred over to U.S. Multi-State Privacy. 

In other words, if your end-user, opted-out from categories for U.S. Privacy (Legacy), that user will continue to be opted-out on U.S. Multi-State Privacy. Same goes for opted-in.

If you ever used authenticated consent for CCPA, you'll have to specify the `ConfigOption.TRANSITION_CCPA_AUTH` option in your configuration to transfer an end-user's opt-in/opt-out preferences. The `ConfigOption.TRANSITION_CCPA_AUTH` option is crucial if you are using AuthId. This way, the SDK will look for authenticated consent within CCPA profiles and carry that over to USNat, even if the user current doesn't have CCPA local data (on a fresh install, for example).

```kotlin
private val spConsentLib by spConsentLibLazy {
    activity = this@YourActivity
    spClient = YourLocalClient()
    config {
        accountId = 22
        propertyId = 39049
        propertyName = "automation-mobile-usnat"
        messLanguage = MessageLanguage.ENGLISH
        +(CampaignType.GDPR)
        +(CampaignType.USNAT to setOf(ConfigOption.TRANSITION_CCPA_AUTH)) // or use addCampaign(SpCampaign(campaignType = CampaignType.USNAT, configParams = setOf(TRANSITION_CCPA_AUTH)))
    }
}
```

```java
private final SpConfig spConfig = new SpConfigDataBuilder()
        .addAccountId(22)
        .addPropertyName("automation-mobile-usnat")
        .addPropertyId(34049)
        .addMessageLanguage(MessageLanguage.ENGLISH)
        .addMessageTimeout(5000)
        .addCampaignsEnv(CampaignsEnv.PUBLIC)
        .addCampaign(new SpCampaign(CampaignType.USNAT, Collections.emptyList(), Set.of(ConfigOption.TRANSITION_CCPA_AUTH)))
        .addCampaign(CampaignType.GDPR)
        .build();
```

Reply:
Thanks!